src/NodeJS/Server.php

```diff
@@ -68,13 +67,13 @@ abstract class Server
     /**
      * Constructor
      *
-     * @param string $host            The server host
-     * @param int    $port            The server port
-     * @param string $nodeBin         Path to NodeJS binary
-     * @param string $serverPath      Path to server script
-     * @param int    $threshold       Threshold value in micro seconds
+     * @param string $host The server host
```

Comment:
Please revert any code reformatting changes, because PR isn't about them.

Reply:
Ups, sorry, I didn't notice these. I left just two to align correctly one comment and one typo.